frontend/src/js/map.js

```diff
@@ -945,9 +955,14 @@ function getRange(data){
         keyCirclesFilter = _.filter(data, keyCircles);
 
         // Add circles to our key array
-        valArray = [ min, keyCirclesFilter[0], keyCirclesFilter[1], max ];        
+        valArray = [ min, keyCirclesFilter[0], max ];        
     }
 
+    if ( max.volume < 5 ){
```

Comment:
No clean way to nest these if statements. Have to keep this check at the end ensuring valArray is populated.